### PR TITLE
T9850: Setup babel cats globally

### PR DESCRIPTION
**Pull request by ugochimobi**
See discussion here: https://meta.miraheze.org/wiki/Community_noticeboard/Archive_31#Babel_-_change_of_configuration_(proposal)

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -244,7 +244,29 @@ function wfShouldEnableSwift( $dbname ) {
 	'wgArticlePlaceholderRepoApiUrl' => [
 		'default' => 'https://www.wikidata.org/w/api.php',
 	],
-
+	
+	// Babel
+	'wgBabelCategoryNames' => [
+		'default' => [
```
Comment (Reception123) [inline LocalSettings.php:250]: This isn't the right default
https://github.com/wikimedia/mediawiki-extensions-Babel/blob/master/extension.json#L66

**Review by Reception123 — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 1 above)
Comment (ugochimobi) [reply LocalSettings.php:250]: > This isn't the right default https://github.com/wikimedia/mediawiki-extensions-Babel/blob/master/extension.json#L66

done

File: LocalSettings.php [hunk 2]
```diff
@@ -244,7 +244,33 @@ function wfShouldEnableSwift( $dbname ) {
 	'wgArticlePlaceholderRepoApiUrl' => [
 		'default' => 'https://www.wikidata.org/w/api.php',
 	],
-
+
```
Comment (Universal-Omega) [inline LocalSettings.php:247]: spacing

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

**Comment by ugochimobi** [comment]
done?

**Comment by Reception123** [comment]
I thought we said since the discussion was on CN we'd apply it globally. If wikis want to opt out they can.

**Comment by ugochimobi** [comment]
The CN thread didn't say anything like that, but I'd support applying it globally. Wdyt?

**Comment by Reception123** [comment]
Well by definition CN is the noticeboard for all Miraheze's wikis and Meta:Community Portal is for Meta matters. Either way, it's quite uncontroversial and if the community didn't want it it could've said so in the CN post.

I recommend making it the default.

**Comment by ugochimobi** [comment]
Done, I think that should be all now.

**Review by Reception123 — APPROVED** [review]

**Pull request merged**